cfme/containers/image.py

```diff
@@ -0,0 +1,56 @@
+from cfme.common import Taggable
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import toolbar as tb
+from cfme.web_ui.menu import nav
+
+from . import list_tbl, cfg_btn, pol_btn
+
+nav.add_branch(
+    'containers_images',
+    {
+        'containers_image':
+        [
+            lambda ctx: list_tbl.select_row_by_cells(
+                {'Name': ctx['image'].name, 'Provider': ctx['provider'].name}),
+            {
+                'containers_image_smartstate_analysis':
+                lambda _: cfg_btn('Perform SmartState Analysis'),
```

Comment:
Doesn't the fleecing button trigger an alert? In that case it would need a parameter in order to work properly. We usually use object methods that navigate to details and then handle clicking and alerts.